src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -1043,6 +1043,7 @@ public function testSubmitNullUsesDateEmptyData($widget, $emptyData, $expectedDa
         $form = $this->factory->create(static::TESTED_TYPE, null, [
             'widget' => $widget,
             'empty_data' => $emptyData,
+            'years' => range(2018, (int) date('Y')),
```

Comment:
This does not seem to be related to the problem described in the issue. Can you revert this change?